deployments/ocis-office/helmfile.yaml

```diff
@@ -98,7 +96,8 @@ releases:
     values:
       - collabora:
           aliasgroups:
-            - host: "http://wopiserver.wopiserver.svc.cluster.local:8880"
+            - host: "wopi-collabora.kube.owncloud.test" # needed because we connect via the ingress to the collaboration service
```

Comment:
Does this still work with the `CoAuthoring` `i-filter`?

Reply:
> Does this still work with the `CoAuthoring` `i-filter`?

Sorry, but I need some more context. What's that?

Reply:
I mean about the mismatch bet the change in there and the new value and the new value here:

https://github.com/owncloud/ocis-charts/blob/275e60f7f968cea2e3ac8712649f0efe41689ce6/deployments/ocis-office/addons/onlyoffice/configmap.yaml#L14

Reply:
> I mean about the mismatch bet the change in there and the new value and the new value here:
> 
> https://github.com/owncloud/ocis-charts/blob/275e60f7f968cea2e3ac8712649f0efe41689ce6/deployments/ocis-office/addons/onlyoffice/configmap.yaml#L14

Collabora is configured to go via the Ingress, therefore the cluster internal hostname is only there as a comment on the allow list:

https://github.com/owncloud/ocis-charts/blob/275e60f7f968cea2e3ac8712649f0efe41689ce6/deployments/ocis-office/helmfile.yaml#L99-L100

We could configure `wopi-onlyoffice.kube.owncloud.test` for OnlyOffice if you prefer this. Then it would be consistent to what we have for Collabora. It was more of a mood to use the wildcard capabilities of OnlyOffice. Actually Collabora should also have those wildcard capabilities according their docs: https://sdk.collaboraonline.com/docs/installation/CODE_Docker_image.html#setting-the-application-configuration-dynamically-via-environment-variables

Reply:
When thinking about it: being more specific in those examples is better. a Change will land asap

Reply:
Don't we still have a deviation between 

https://github.com/owncloud/ocis-charts/blob/e5ba13ed533da848fb5e08c1ce7d4823febfde67/deployments/ocis-office/addons/onlyoffice/configmap.yaml#L14

and 

https://github.com/owncloud/ocis-charts/blob/275e60f7f968cea2e3ac8712649f0efe41689ce6/deployments/ocis-office/helmfile.yaml#L66

?

Reply:
> Don't we still have a deviation between

Yes, but it's fine:

> https://github.com/owncloud/ocis-charts/blob/e5ba13ed533da848fb5e08c1ce7d4823febfde67/deployments/ocis-office/addons/onlyoffice/configmap.yaml#L14

This only configures which WOPI servers OnlyOffice is allowed to contact. In this case we showcase connectiong to the wopi server cluster internally, while we showcase with Collabora how to connect via Ingress.

> https://github.com/owncloud/ocis-charts/blob/275e60f7f968cea2e3ac8712649f0efe41689ce6/deployments/ocis-office/helmfile.yaml#L66

This is the ingress for OnlyOffice. It needs to be "public" so that users can access it ("via the internet"). Please note that also Collabora has such a Ingress.